scripts/pc2sandbox_interactive.sh

```diff
@@ -308,8 +361,9 @@ TIMELIMIT_US=$((TIMELIMIT * 1000000))
 REPORT Setting cpu limit to $TIMELIMIT_US microseconds "("ulimit -t $TIMELIMIT ")"
 ulimit -t $TIMELIMIT
 
-REPORT Setting maximum user processes to $MAXPROCS + whatever the user is currently using
-ulimit -u $((MAXPROCS+`ps -T -u $USER | wc -l`))
+MAXPROCS=$((MAXPROCS+`ps -T -u $USER | wc -l`))
+REPORT Setting maximum user processes to $MAXPROCS 
+ulimit -u $MAXPROCS
 
 # Remember wall time when we started
 starttime=`GetTimeInMicros`
```

Comment:
With the above command being located here, the user will be charged for all the time it takes to execute the statements between here and the `setsid taskset` command (below).  If REPORT and/or REPORT_DEBUG are enabled, that could be significant. it seems like this statement  belongs _right before_ the call to `setsid taskset`...

Reply:
This is wall time, not CPU time.  We use ulimit -t to limit CPU time, so submission has it's own "cpu time" separate from script (and every other process).  So, yes, the wall time may be off slightly, but it won't cause the submission to be terminated.